src/containers/PayStrings/test/index.test.tsx

```diff
@@ -16,7 +17,7 @@ jest.mock('../../../rippled', () => {
   }
 })
 
-const mockGetPayString = getPayString
+const mockGetPayString = getPayString as Mock
```

Comment:
Why as

Reply:
So that typescript would not complain about calling things like `resetMock`